torchvision/transforms/functional_tensor.py

```diff
@@ -191,6 +191,14 @@ def adjust_contrast(img: Tensor, contrast_factor: float) -> Tensor:
 
 
 def adjust_hue(img: Tensor, hue_factor: float) -> Tensor:
+    """Adjust hue of an RGB image (batch of images)
+    Args:
+        img (Tensor): a Tensor representing a 2D RGB image (CHW) or a batch of
+            2D RGB images (NCHW).
+            Note that the pixel values of the image has to be non-negative, so
+            it does not work if you normalize your image to an interval with
+            negative values like [-1, 1] before using this function.
+    """
```

Comment:
@drbeh These methods are currently considered private and that's why they are undocumented. 

The public implementations are on `functional.py` and `transforms.py`, so it's worth updating the documentation there.